.mailmap

```diff
@@ -38,6 +40,7 @@ Trent Nelson <trent@trent.me>
 ccasey <ccasey@651a555e-23ca-0310-84fe-ca9f7c59d2ea>
 conda-bot <ad-team+condabot@anaconda.com> Conda Bot <18747875+conda-bot@users.noreply.github.com>
 conda-bot <ad-team+condabot@anaconda.com> conda bot <18747875+conda-bot@users.noreply.github.com>
+dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
 dmartin <dmartin@651a555e-23ca-0310-84fe-ca9f7c59d2ea>
 dpeterson <dpeterson@651a555e-23ca-0310-84fe-ca9f7c59d2ea>
```

Comment:
should we add proper names/emails for ccasey, dmartin, and dpeterson?

Reply:
I can't even find the commits they have made, honestly.

Reply:
found via `git log --author=ccasey` etc.:

- ccasey: https://github.com/conda/menuinst/commit/19b3137b9f55bb8bc00303e330bcf1698fc9f7d1
- dmartin: https://github.com/conda/menuinst/commit/34842b9008383219ca05eabdb2b7b73edefe6746
- dpeterson: https://github.com/conda/menuinst/commit/6a7251092b65cad401b431fafe815242e15a59ef